cmd/juju/model/destroy.go

```diff
@@ -101,22 +100,19 @@ See also:
 `
 
 var destroyModelMsg = `
-WARNING! This command will destroy the %q model.
-`[1:]
-
-var destroyIAASModelMsgWithDetails = `
-WARNING! This command will destroy the %q model.
-This includes %d machine(s), %d application(s), data and other resources.
-
-The following machines will be destroyed: %s
-`[1:]
-
-var destroyCAASModelMsgWithDetails = `
-WARNING! This command will destroy the %q model.
-This includes %d container(s), %d application(s), data and other resources.
-
-The following containers will be destroyed: %s
-`[1:]
+WARNING! This command will destroy the %q model and all it's resources`[1:]
+
+var destroyModelMsgDetails = `
+{{- if gt .MachineCount 0}}
+ - {{.MachineCount}} {{if .IsCaaS}}container{{else}}machine{{end}}{{if gt .MachineCount 1}}s{{end}} will be destroyed
+  - {{if .IsCaaS}}container{{else}}machine{{end}} list:{{range .MachineIds}} "{{.}}"{{end}}
+ - {{.ApplicationCount}} application{{if gt .ApplicationCount 1}}s{{end}} will be removed
+ {{- if gt .ApplicationCount 0}}
+  - application list:{{range .ApplicationNames}} "{{.}}"{{end}}
```

Comment:
It seems this doesn't quite work
```
$ juju destroy-model m
WARNING! This command will destroy the "m" model and all it's resources
 - 3 machines will be destroyed
  - machine list: "1 (juju-12f00f-1)" "2 (juju-12f00f-2)" "3 (juju-12f00f-3)"
 - 3 applications will be removed
  - application list:
 - 0 filesystem and 0 volume will be destroyed
```
I'm not exactly sure where the data is being lost for this tbh

Reply:
cannot reproduce this issues